### ISSUE TITLE
Let's discuss evolving wordpress.org/gutenberg

### ISSUE THREAD

**Issue by annezazu** [WordPress/wporg-main-2022#379]
With WordPress Playground allowing folks to have WordPress in a browser, the WordPress.org website redesign push, and the [introduction of a new Blocks page](https://github.com/WordPress/wporg-main-2022/issues/318), let's revisit the role and approach of /gutenberg. Originally, it was implemented to share a vision of the project and give people a hands on way to explore how blocks work on the page. WordPress Playground covers the hands on aspect nicely and Blocks helps share the vision of all that can be done within and outside of WordPress. 

**Where does this leave /Gutenberg? Let's discuss here and see what makes sense.**

**Comment by annezazu** [comment]
_Let me know if this wasn't the right place to start this discussion._

In my mind, I think /Gutenberg should become more about the overall vision of the project, how to get involved, list of key milestones, and move away from having it be a way for folks to use blocks. There's also a need for folks to understand that [Gutenberg is the project name and plugin rather than the Editor experience](https://wordpress.org/news/2021/04/become-an-early-adopter-with-the-gutenberg-plugin/) and this page doesn't help that cause.

**Comment by alexstine** [comment]
I actually really like this demo page. It is isolated to the editor and I'd rather not replace that with an iFrame experience.

**Comment by annezazu** [comment]
Thanks for chiming in, Alex! I can see a world where we keep the interactive nature to it too. Right now, I'm mainly keen to figure out the content between these two pages and resolve that hurtle. We might be in a situation where it makes more sense for this page to be overhauled in the way /blocks is and for /blocks to become more of a deep dive about all of the functionality blocks provide, including block themes, patterns, block style variations, custom blocks, interactivity features (in the future), and more.

**Comment by tobifjellner** [comment]
The demonstration of the block editor is a much smaller load for a web browser than a whole WordPress simulation. So it's useful to have a separate demo if the block editor. But it should live on s different page, since Gutenberg is a project much bigger than just the editor.
We could use /gutenberg to describe the four project phases and current status, and inviting to take part in the discussion, testing and development.

**Comment by peteringersoll** [comment]
Anything we can do to clarify the what the Gutenberg project is vs. the editor itself would be helpful. As a meetup organizer, it's a challenge to try and help folks understand WordPress even the terms used cause uncertainty. And I hope this can be communicated through all channels (and all levels, up to the top).

**Comment by annezazu** [comment]
>Anything we can do to clarify the what the Gutenberg project is vs. the editor itself would be helpful.

Agreed. That's part of my hope in distinguishing the two pages. For example, in your case, I'd imagine /blocks being more useful for you in the future.

**Comment by ndiego** [comment]
> In my mind, I think /Gutenberg should become more about the overall vision of the project, how to get involved, list of key milestones, and move away from having it be a way for folks to use blocks. There's also a need for folks to understand that [Gutenberg is the project name and plugin rather than the Editor experience](https://wordpress.org/news/2021/04/become-an-early-adopter-with-the-gutenberg-plugin/) and this page doesn't help that cause.

Fully agree with this. I think there is an opportunity for `/gutenberg` to be an actual page that provides a comprehensive overview of the project itself. 

That said, I do think there is value in having a "demo page" where folks can play around in the Editor via the browser, but perhaps this could be located at a different URL. This demo page should then run the latest WordPress default theme and could feature a short intro section explaining the page, but then patterns from the current theme. This could be a fun way of keeping the demo content fresh while allowing people to test out the latest and greatest of WordPress. The current demo feels a bit dated.

I believe this was explored somewhere (can't find the link), but this demo should also be "framed" by the WordPress.org primary navigation and footer. It should feel part of the site IMO.

**Comment by alexstine** [comment]
> I believe this was explored somewhere (can't find the link), but this demo should also be "framed" by the WordPress.org primary navigation and footer. It should feel part of the site IMO.

This is the one part I really want to avoid. iFrames are not great for accessibility and an iFrame inside an iFrame really isn't great for accessibility. I wouldn't mind having a "Back to WordPress.org" link but the page needs to remain isolated like the real experience would be in WP Admin.

**Comment by piermario** [comment]
I agree with all the comments above. /gutenberg could be used to provide an overview of the project, which could include a timeline that shows where we are right now and what's ahead. The demo should be its own page, both for accessibility and convenience. I often used the current /gutenberg page to give a quick demo of the block editor and it's always been very handy. 

What if the new demo URL was /gutenberg/demo or /gutenberg/try ?

**Comment by annezazu** [comment]
I chatted with @mtias about this to get some project leadership direction and insight. He kindly clarified that /blocks is meant to be about how WordPress uses blocks, the block directory, and an introduction for developers who want to build with blocks. In contrast, the /gutenberg page should be more about devrel, the architecture of gutenberg, and its use outside of just WordPress. The latter aligns nicely around what Riad has been up to with Gutenberg as a framework resources: https://github.com/WordPress/gutenberg/issues/53874 What's being worked on here can likely take over what's in place currently. 

With the above in mind, let's start moving in these directions in future iterations. I'm going to loop back on /blocks and, in time, we can evolve /gutenberg in line with the above.
